server/lib/front.go

```diff
@@ -6,11 +6,13 @@ import (
 	"log"
 	"time"
 
-	proto "../proto"
-	"./packets"
-	"./assets"
-	"./models"
+	proto "github.com/umegaya/channer/server/proto"
+	"github.com/umegaya/channer/server/lib/packets"
+	"github.com/umegaya/channer/server/lib/assets"
+	"github.com/umegaya/channer/server/lib/models"
```

Comment:
__[gometalinter]__
* [gotype] could not import github.com/umegaya/channer/server/lib/models (can't find import: github.com/umegaya/channer/server/lib/models)